x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/details_step/details_step_details.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, Fragment } from 'react';
+import { i18n } from '@kbn/i18n';
+import {
+  EuiButtonEmpty,
+  EuiDescriptionList,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+} from '@elastic/eui';
+import { State } from '../../../analytics_management/hooks/use_create_analytics_form/state';
+import { ANALYTICS_STEPS } from '../../page';
+
+export interface ListItems {
+  title: string;
+  description: string | JSX.Element;
+}
+
+export const DetailsStepDetails: FC<{ setCurrentStep: any; state: State }> = ({
+  setCurrentStep,
+  state,
+}) => {
+  const { form } = state;
+  const { description, jobId, destinationIndex } = form;
+
+  const detailsFirstCol: ListItems[] = [
+    {
+      title: i18n.translate('xpack.ml.dataframe.analytics.create.configDetails.jobId', {
+        defaultMessage: 'Job id',
```

Comment:
We normally use uppercase `ID` for `Job ID`. The `details_step_form.tsx ` for example uses `Job ID`.